### PR TITLE
fix: [C4R-1209] contact person for an ong is now jsonb instead of secondary entity

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
[C4R-1209](https://wearetribus.atlassian.net/browse/C4R-1209)

### CODE REVIEW

File: backend/src/modules/organization/dto/update-organization-general.dto.ts [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 import { CreateOrganizationGeneralDto } from './create-organization-general.dto';
 
 export class UpdateOrganizationGeneralDto extends PartialType(
-  OmitType(CreateOrganizationGeneralDto, ['contact']),
+  CreateOrganizationGeneralDto,
 ) {
-  /* Organization contact person */
   @IsOptional()
-  @Type(() => UpdateContactDto)
-  @ValidateNested()
```
Comment (radulescuandrew) [inline backend/src/modules/organization/dto/update-organization-general.dto.ts:13]: E OK fara ValidateNested?

File: frontend/src/services/organization/OrganizationFormDataMapper.service.ts [hunk 2]
```diff
@@ -32,7 +32,6 @@ export const mapOrganizationGeneralToFormDara = (
   organizationGeneralPayload.append(`${organizationGeneralKey}[countyId]`, county?.id.toString());
   organizationGeneralPayload.append(`${organizationGeneralKey}[cityId]`, city?.id.toString());
 
-  // map contact
   const organizationWithContactPayload = mapEntityToFormData(
```
Comment (radulescuandrew) [inline frontend/src/services/organization/OrganizationFormDataMapper.service.ts:35]: De asta mai e nevoie? @birloiflorian

File: backend/src/modules/organization/dto/update-organization-general.dto.ts (hunk 1 above)
Comment (OlteanuCosmin) [reply backend/src/modules/organization/dto/update-organization-general.dto.ts:13]: Readaugat ValidateNested. Este intradevar nevoie de el. Initial am crezut ca imi cauza o problema la update.

File: frontend/src/services/organization/OrganizationFormDataMapper.service.ts (hunk 2 above)
Comment (radulescuandrew) [reply frontend/src/services/organization/OrganizationFormDataMapper.service.ts:35]: @OlteanuCosmin

**Review by radulescuandrew — APPROVED** [review]

**Pull request merged**